cmake/compiler.cmake

```diff
@@ -40,6 +40,8 @@ if(MSVC)
 			"/OPT:REF"
 			"/OPT:ICF"
 			"/INCREMENTAL:NO"
+			# Ignore warnings about PDBs not being found (happens e.g. for vc140)
+			"/ignore:4099"
```

Comment:
More specifically, this happens because vcpkg's [portfile](https://github.com/microsoft/vcpkg/blob/master/ports/icu/portfile.cmake) doesn't install the PDB file(s).